Examples api is broken

### **Describe the bug**
```

error[E0026]: variant `MenuEvent` does not have a field named `window_id`
    --> core/tauri-runtime-wry/src/lib.rs:1505:7
     |
1505 |       window_id,
     |       ^^^^^^^^^ variant `MenuEvent` does not have this field

error[E0026]: variant `MenuEvent` does not have a field named `window_id`
    --> core/tauri-runtime-wry/src/lib.rs:1521:7
     |
1521 |       window_id: _,
     |       ^^^^^^^^^ variant `MenuEvent` does not have this field


```
### **To Reproduce**
Steps to reproduce the behavior:

Steps described in readme to compile and run examples/api

### **Platform and Versions (required):**

Macos
yarn info v1.22.10
rustc 1.52.1 (9bc8c42bb 2021-05-09)  (1.53 also used)

tauri info:
```
 app:spawn [sync] Running "cargo build --release" +0ms

   Compiling tauri-cli v1.0.0-beta.4 ( *** tauri/tooling/cli.rs)
    Finished release [optimized] target(s) in 11.58s
 app:spawn Running " *** tauri/tooling/cli.rs/target/release/cargo-tauri tauri info" +12s

Operating System - Mac OS, version 11.4.0 X64

Node.js environment
  Node.js - 16.3.0
  @tauri-apps/cli - 1.0.0-beta.5
  @tauri-apps/api - 1.0.0-beta.4

Global packages
  npm - 7.15.1
  yarn - 1.22.10

Rust environment
  rustc - 1.53.0
  cargo - 1.53.0

App directory structure
/node_modules
/public
/src-tauri
/src

App
  tauri.rs - path:"../../../core/tauri" [1.0.0-beta.4] (no lockfile)
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: asset: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../public
  devPath - ../public
  framework - Svelte
  bundler - Rollup

```

--- a/examples/api/README.md
+++ b/examples/api/README.md
@@ -18,6 +18,7 @@ go to root of the tauri repo and run
 
 Linux / Mac:
 ```
+# choose to install node cli (1)
 sh .scripts/setup.sh
 ```
 

